models/vyos/ntp/vyos_ntp.yaml

```diff
@@ -0,0 +1,61 @@
+---
+
+GENERATOR_VERSION: '1.0'
+ANSIBLE_METADATA: |
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+
+NETWORK_OS: vyos
+RESOURCE: ntp
+COPYRIGHT: Copyright 2021 Red Hat
+
+
+DOCUMENTAION: |
+  module: vyos_ntp
+  version added:
+  short_description:  Manages ntp modules of Vyos
+  description: This module manages ntp configuration on devices running Vyos
+  author: Varshitha Yataluru (@vyatalur)
+  options:
+    config:
+        description: List of configurations for ntp module
+        type: list
+        elements: dict
+        suboptions:
+            allow-clients:
+                description: Network Time Protocol (NTP) server options
+                type: str
+                suboptions:
+                    address:
+                        description: IP address
+                        type: str
+                        choices:
```

Comment:
please use ref for similar data in your model it would help you optimise your model.
example : https://github.com/ansible-network/resource_module_models/blob/1617db5d498bb75639f55d91419f0f6fe26c8cf4/models/ios/logging_global/ios_logging_global.yaml#L27